wiki/ticket-splitting-v1.6.md

```diff
@@ -0,0 +1,145 @@
+
+dcrwallet v1.6 also does a one-way migration of the wallet file and older versions will not work with it. { correct? }
+
+## Will it break? When?
+
+Yes. The software stack that currently supports ticket splitting will stop working around **May 7, 2021** when v1.5 will stop syncing.
+
+Legacy VSPs should still vote any split tickets bought before that date, but it will not be possible to purchase new split tickets with v1.5, because it will not sync. More technical details [here](https://chat.decred.org/#/room/!zefvTnlxYHPKvJMThI:decred.org/$OsFGKqqDYnWER-HPuoaqDDvO0b4FzHanXU7UuEntCZo).
+
+## What is ticket splitting again?
+
+Ticket splitting allows to buy a fraction of a ticket with at least 5 DCR. Participants get:
+
+- a _chance_ to cast a consensus vote, proportional to one's input (e.g. funding 60% of a ticket gives a 60% chance to vote with it)
+- a _fraction_ of the voting reward, proportional to one's input
+- Politeia voting rights are granted to the biggest contributor
+
+See more in the [tutorials](https://decredvoting.com/ticketsplitting).
+
+## Why does it not work with v1.6?
+
+The split ticket buyer program "logs in" to dcrwallet in a certain way. This way has [changed](https://github.com/decred/dcrwallet/pull/1867) in dcrwallet v1.6 to improve security and support the new vspd model and possibly other features. A bunch of software talking to dcrwallet (Decrediton, politeiavoter, dcrtime, etc) was updated to this change but the split ticket buyer program [was not](https://github.com/matheusd/dcr-split-ticket-matcher/issues/77).
+
+## Why does it not work with vspd?
+
+The new accountless VSP model is very different from the "legacy" VSP model. It offers [huge benefits](https://blog.decred.org/2020/06/02/A-More-Private-Way-to-Stake/) like private staking and better UX, but existing ticket splitting software does not work with it.
+
+A whole new matching protocol, client and server are [required](https://www.reddit.com/r/decred/comments/lj2her/og_ticket/gnj3zqq/) to split the tickets with vspd. Put simply, it requires a lot of work.
+
+## Why don't you guys fix it?
+
+There is strong indication that the demand for ticket splitting is not enought to justify pulling our scarce dev resources away from other important work.
+
+Over almost 3 years 2,679 split tickets were purchased at an average rate of 81/month, meaning that roughly 0.2% of tickets are being split. Largest individual month had 127 split ticket purchases (2020-Mar).
+
+In addition, we have this data [gathered](https://www.reddit.com/r/decred/comments/m547to/not_focusing_on_split_tickets_will_affect_us/gr0in3e/) from the blockchain:
+
+- split ticket purchases by month [chart](https://pasteboard.co/JSJHHdY.png)
+- split ticket purchases by the number of particpants [chart](https://pasteboard.co/JSJI0f0.png)
```

Comment:
If at all possible, I'd recommend bringing in these pics locally to this repo and linking to it here (vs on pasteboard which could go away at any time).

Reply:
Done, thanks for pointing it out.

I try to keep text repos super light by not adding binaries, but turns out I can put them into an orphan branch and link there. It is much better than relying on external sources, but is still flexible for low traffic scenarios (e.g. fetch only text with `git clone --single-branch ...` and then fetch the heavy branch with binaries when feasible).